src/app/footer/footer-tags/footer-tags.component.html

```diff
@@ -0,0 +1,15 @@
+<div class="footer-tags_container">
```

Comment:
Beautify your templates - you don't need so mane unneeded empty lines.

Reply:
fixed